docs/guides/key-manager/upgrade-lsp6.md

```diff
@@ -0,0 +1,244 @@
+---
+sidebar_label: 'Upgrade LSP6 Key Manager'
+sidebar_position: 4
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Upgrade the LSP6 Key Manager
+
+:::info Requirements
+
+You will need a Universal Profile that you can control via its KeyManager to follow this guide. <br/>
+If you don't have a Universal Profile yet, follow our previous guide [**Create a Universal Profile**](../universal-profile/create-profile.md) or look at the [_lsp-factory.js_](../../tools/lsp-factoryjs/deployment/universal-profile.md) docs.
+
+:::
+
+In this guide, we will learn how upgrade the LSP6 Key Manager of your Universal Profile to the latest version available.
+
+By the end of this guide, you will know:
+
+- How to deploy a new LSP6 Key Manager with the last updates.
+- How to transfer ownership of your Universal Profile through the old LSP6 Key Manager.
+- How to accept ownership of your Universal Profile through the new LSP6 Key Manager
+
+## Setup
+
+```shell
+npm install @lukso/lsp-smart-contracts, web3
+```
+
+## Step 1 - Create `constants.js` file and save the needed constants
+
+Create a `constants.js` file with the following constants
+
+- PRIVATE_KEY = private key of a conroller address
+- keyManagerAddress = address of the current LSP6 Key Manager
+- universalProfileAddress = address of your Universal Profile
+  This controller must have CHANGEOWNER permission
+
+After this first step you can save & close `constants.js`.
+
+```js
+export const PRIVATE_KEY = '0x...';
+export const universalProfileAddress = '0x...';
+export const keyManagerAddress = '0x...';
+```
+
+## Step 2 - Create `upgrade-lsp6.js` and get the imports
+
+Create a file with the name `upgarde-lsp6.js` and you should have the following imports on the top of the file.
+
+```js
+import LSP6KeyManager from '@lukso/lsp-smart-contracts/artifacts/LSP6KeyManager.json' assert { type: 'json' };
+import {
+  PRIVATE_KEY,
+  keyManagerAddress,
+  universalProfileAddress,
+} from './constants.js';
+import Web3 from 'web3';
```

Comment:
Add spaces so it is more readable and users can differentiate between internal/external 
```suggestion
} from './constants.js';

import Web3 from 'web3';
```